Dockerfile

```diff
@@ -18,17 +18,18 @@ ENV DJANGO_SETTINGS_MODULE=basket.settings
 # Install app
 COPY requirements/* /app/requirements/
 
-# TODO: split out a separate dev image from the prod image and only install scoped deps
-# RUN pip install --require-hashes --no-cache-dir -r requirements/prod.txt
-RUN pip install --require-hashes --no-cache-dir -r requirements/dev.txt
+# The setuptools install is needed for pyfxa (currently v0.7.7) which calls `pkg_resources`,
+# and Python 3.12 no longer adds setuptools by default to the venv.
+RUN pip install -U setuptools && \
```

Comment:
setuptools is needed at "runtime" in basket b/c of pyfxa. pip-tools only installs it so it can compile the requirements but it doesn't get into the venv. So I still need this.